plugins/modules/database/misc/odbc.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, John Westcott <john.westcott.iv@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: odbc
+author: "John Westcott IV (@john-westcott-iv)"
+version_added: "2.8"
+short_description: Execute SQL via ODBC
+description:
+    - Read/Write info via ODBC
+options:
+    dsn:
+      description:
+        - The connection string passed into ODBC
+      required: True
+      type: str
+    query:
+      description:
+        - The SQL query to perform
+      required: True
+      type: str
+    params:
+      description:
+        - Parameters to pass to the SQL squery
+      type: list
+      elements: str
+
+requirements:
+  - "python >= 2.6"
+  - "pyodbc"
+'''
+
+EXAMPLES = '''
+- name: Set some values in the test db
+  odbc:
+    dsn: "DRIVER={ODBC Driver 13 for SQL Server};Server=db.ansible.com;Database=my_db;UID=admin;PWD=password;"
+    query: "Select * from table_a where column1 = ?"
+    params:
+      - "value1"
+'''
+
+RETURN = '''
+results:
+    description: List of dicts containing selected rows, likley empty for DDL statements.
+    returned: success
+    type: list
```

Comment:
```suggestion
    type: list
    elements: ...
```
Always use `elements: xxx` with `type: list` if possible.